wom/src/main/scala/wom/types/WomOptionalType.scala

```diff
@@ -0,0 +1,58 @@
+package wom.types
+
+import spray.json.JsNull
+import wom.values.{WomOptionalValue, WomValue}
+
+import scala.util.Try
+
+case class WomOptionalType(memberType: WomType) extends WomType {
+  /**
+    * Method to be overridden by implementation classes defining a partial function
+    * for the conversion of raw input values to specific implementation class value types.
+    * i.e.  `WdlBooleanType` should define a partial function that knows how to
+    * construct `WdlBoolean`s for inputs of supported types and contents.  Values for which
+    * the partial function is not defined are assumed to not be convertible to the target type.
+    */
+  override protected def coercion: PartialFunction[Any, WomValue] = {
+
+    // It's safe to box up values implicitly:
+    case wdlValue: WomValue if memberType.equals(wdlValue.wdlType) => WomOptionalValue(wdlValue)
```

Comment:
I bet there'll be a lot of these to fix up as we see them... And I just saw this one.

Reply:
now fixed